lib/renderer.js

```diff
@@ -1261,10 +1261,11 @@
         // 10. The layer is not hidden.
         var layer = component.layer,
             layerComp = component.comp,
-            hasComplexTransform = (layer || layerComp) &&
-                ((component.hasOwnProperty("scale") && component.scale % 1 !== 0) ||
+            hasComplexTransform = ((layer && ((component.hasOwnProperty("width") ||
```

Comment:
should the first condition continue to start with `layer || layerComp` to recognize layerComps with absolute h/w params?